src/index.ts

```diff
@@ -11,6 +13,10 @@ import models from './models';
 import Example from './models/Example';
 
 const config = {};
+// configure loggin
+if (process.env.SENTRY_DSN !== 'false') {
```

Comment:
Would you export a `Raven` instance from the config file then? Or what is the idea?